views/blp/license_server_fact.view.lkml

```diff
@@ -223,6 +223,14 @@ view: license_server_fact {
     hidden: no
   }
 
+  dimension: customer_name_unlinked {
+    label: " Customer Name (Unlinked)"
```

Comment:
What is this field for?

Reply:
The Customer name field is linked with Customer ID for a Salesforce Report, when I try to use the field as is, it brings Customer ID along with it. I need to create this new field, essentially the same field, which a different name, and use this.

Reply:
Thanks! Can you help add this information to the description? Perhaps

```
Customer name. Created to separate Name and ID fields, which are both included in the same Salesforce customer field.
```